pkg/cmd/types.go

```diff
@@ -114,20 +110,22 @@ func (i ignoredFields) Contains(field string) bool {
 var defaultIgnored = ignoredFields{"password", "token", "url", "uri", "name", "type", "id"}
 
 //Convert a kubernetes secret to a mobile.ServiceConfig
-func (dsc defaultSecretConvertor) Convert(id string, params map[string]string, configType string) (*ServiceConfig, error) {
+func (dsc defaultSecretConvertor) Convert(secret v1.Secret) (*ServiceConfig, error) {
+	params := secret.Data
 	config := map[string]interface{}{}
 	headers := map[string]string{}
 	for k, v := range params {
+		strV := string(v)
 		if !defaultIgnored.Contains(k) {
-			if configType == "json" && k == "config" {
```

Comment:
@maleck13 should we remove configType from annotations then? Looks like we dont need it

Reply:
Actually think this is fine, should allow a user to dump some strings into a map